src/create/Form.js

```diff
@@ -4,32 +4,54 @@ import styled from 'styled-components'
 const StyledForm = styled.form`
   display: grid;
   grid-gap: 12px;
-  grid-template-rows: 40px 60px 40px 48px;
+  grid-template-rows: 80px 80px 80px 48px;
+`
+const StyledInputArea = styled.section`
+  display: grid;
+  grid-auto-flow: rows;
+`
+
+const StyledLabel = styled.label`
+  display: block;
+  padding: 4px;
 `
 
 export default function Form({ data, onSubmit, onInputChange }) {
   return (
     <StyledForm onSubmit={onSubmit}>
-      <input
-        onChange={onInputChange}
-        value={data.title}
-        type="text"
-        placeholder="Kampagnenname"
-        name="title"
-      />
-      <input
-        onChange={onInputChange}
-        value={data.brand}
-        placeholder="Brand"
-        name="brand"
-      />
-      <input
-        onChange={onInputChange}
-        value={data.tags}
-        type="text"
-        placeholder="Tags"
-        name="tags"
-      />
+      <StyledInputArea>
+        <StyledLabel for="kampagnenname__input">Kampagne</StyledLabel>
```

Comment:
was genau macht for=...?

Reply:
for stellt die verbindung zwischen input und label her und bezieht sich auf die input id=